package.json

```diff
@@ -6,8 +6,9 @@
   "license": "MIT",
   "dependencies": {
     "@expo/spawn-async": "^1.2.8",
+    "expo-constants": "^6.0.0",
```

Comment:
we should make this `"expo-constants": "*"` so we don't have to update the package every release. this way we can just use whatever people have installed already. if the constants change in the future (they have been relatively stable) we'll need to update the library